saleor/graphql/account/resolvers.py

```diff
@@ -106,7 +106,8 @@ def resolve_address_validation_rules(
 
 def resolve_payment_sources(user: models.User):
     stored_customer_accounts = (
-        (gtw, fetch_customer_id(user, gtw)) for gtw in gateway.list_gateways()
+        (gtw["name"], fetch_customer_id(user, gtw["name"]))
```

Comment:
Let's add some tests for this account.

Reply:
added query test, and modify payment test so it doesn't mock `list_gateways()`